packages/utils/src/throwError.ts

```diff
@@ -0,0 +1,4 @@
+// Tiny helper for throwing errors from expressions
```

Comment:
nit: is this even worth? it's just 3 chars shorter and you need to import it

Reply:
feel free to resolve if you think it's ok

Reply:
> 3 chars shorter

than what? The reason is to be able to do things like `const item = maybeItem ?? throwError('missing!')` which unfortunately isn't possible with simple throwing. But yes, the helper is really tiny. I just didn't want to repeat it whenever I want to use it.

Reply:
I think this should work to `const item = maybeItem ?? throw new Error('missing!')`, but yea in the end wrapper is not bad idea, people usually forget to add `new` etc. 

Sorry for bothering, resolving.